frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -60,30 +278,102 @@ const Dashboard: FunctionComponent<DashboardParams> = ({
   if (isSignedOut) {
     history.push("/login");
   }
+  const [screenWidth, setScreenWidth] = useState(window.innerWidth);
+
+  useEffect(() => {
+    const resizeScreen = () => {
+      setScreenWidth(window.innerWidth);
+    };
+    window.addEventListener("resize", resizeScreen);
+    return () => {
+      window.removeEventListener("resize", resizeScreen);
+    };
+  }, []);
+
   useEffect(() => {
     if (!currentUser) {
       getUser({});
     }
   }, [currentUser, getUser]);
+
   if (!currentUser) {
     return <h1>Loading</h1>;
   }
+
   return (
-    <Layout>
+    <Layout shouldAddPadding={false}>
       <>
-        Dashboard
-        <SignOutButton onClick={signOut}>sign out</SignOutButton>
-        <PurpleButton
-          text={getTitleOfButton(currentUser)}
-          onClick={() =>
-            history.push(
-              currentUser.onAssessment
-                ? "/assessments"
-                : "/interventions"
-            )
-          }
-          disabled={currentUser.sessionId === 9}
-        />
+        <DashboardContainer>
+          <MenuContainer>
+            <ProfilePicture
+              src={
+                "https://firebasestorage.googleapis.com/v0/b/vocab-buddy-53eca.appspot.com/o/dajin.png?alt=media&token=933c72b9-afaf-407b-b978-bfd2c3b4e155"
+              }
+            />
+            <TitleText>hi name!</TitleText>
+            <MenuButtonContainer></MenuButtonContainer>
+
+            {screenWidth > 600 && (
+              <SignOutButton onClick={signOut}>log out</SignOutButton>
+            )}
+          </MenuContainer>
```

Comment:
We probably want the sign out button no matter what, so I would not worry about the screen width here.

Reply:
I put this in because by default, the sign out button is hidden on the mobile view. It shows up once that menu is expanded. Should I take this out?

Reply:
Nvm keep it then!